Nodes/Brev.c

```diff
@@ -660,7 +661,7 @@ int updatenextletter(int user) {
 }
 
 void sparabrev(void) {
-  BPTR fh, lock = NULL;
+  BPTR fh, lock = 0;
```

Comment:
I have not seen it before, but VBCC seems to be very strict about NULL pointer vs 0, and I guess technically it's correct.